test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -72,19 +68,21 @@ namespace integration_framework {
       auto perm = static_cast<shared_model::interface::permissions::Role>(i);
       all_perms.set(perm);
     }
-    auto genesis_tx =
-        shared_model::proto::TransactionBuilder()
-            .creatorAccountId(kAdminId)
-            .createdTime(iroha::time::now())
-            .addPeer("0.0.0.0:50541", key.publicKey())
-            .createRole(kDefaultRole, all_perms)
-            .createDomain(kDefaultDomain, kDefaultRole)
-            .createAccount(kAdminName, kDefaultDomain, key.publicKey())
-            .createAsset(kAssetName, kDefaultDomain, 1)
-            .quorum(1)
-            .build()
-            .signAndAddSignature(key)
-            .finish();
+    auto genesis_tx = shared_model::proto::TransactionBuilder()
+                          .creatorAccountId(kAdminId)
+                          .createdTime(iroha::time::now())
+                          .addPeer("0.0.0.0:50541", key.publicKey())
+                          .createRole(kAdminRole, all_perms)
+                          .createRole(kDefaultRole, {})
```

Comment:
That is because before this change DefaultRole was the only admin's role and he needed all the permissions. As a side effect, that also gave all permissions to every user created afterwards. Now I have assigned admin a separate role, and the users have to be granted their permissions explicitly. This reduces the risk of undeliberately leaving extra permissions for users.